components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, EventEmitter, Input, Output, OnChanges, OnInit, SimpleChanges } from '@angular/core';
```

Comment:
This file, which was previously projects-dropdown, is virtually unchanged, except for changing "projects" to "resources".